db/migrate/20221018151725_add_post_counter_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostCounterToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :PostCounter, :integer
```

Comment:
- [ ] [OPTIONAL] Kindly notice that it is better to write the name of the column `snake_case` not `UpperCamelCase` as all the columns are like this.

```ruby
add_column :users, :posts_counter, :integer
```